### PR TITLE
Return electron count position as a 1D array index

### PR DESCRIPTION
**Pull request by cjh1**
This is a simpler representation rather than have to pass around
std::pair.

### CODE REVIEW

File: examples/electron.ipynb [hunk 1]
```diff
@@ -51,7 +51,9 @@
     "# Just sum the events for now\n",
     "for frame in frame_events:\n",
     "    for pos in frame:\n",
-    "        image[pos[0]][pos[1]] += 1\n"
+    "        row = pos // 576\n",
```
Comment (psavery) [inline examples/electron.ipynb]: Can we figure out `576` here programmatically?

Comment (cjh1) [reply examples/electron.ipynb]: Not real, its just the physical size of the detector.

Comment (psavery) [reply examples/electron.ipynb]: Okay. Yeah, I kind of wish that we could. But it's okay.

**Comment by psavery** [comment]
Looks great to me!

**Pull request merged**